json-endpoints/jetpack/class.jetpack-json-api-maybe-auto-update-endpoint.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+class Jetpack_JSON_API_Maybe_Auto_Update_Endpoint extends Jetpack_JSON_API_Endpoint {
+	// POST /sites/%s/maybe_auto_update
+	protected $needed_capabilities = array( 'update_core', 'update_plugins', 'update_themes' );
+
+	protected $update_results = array();
+
+	protected function result() {
+		add_action( 'automatic_updates_complete', array( $this, 'get_update_results' ), 100, 1 );
+
+		include_once( ABSPATH . '/wp-admin/includes/admin.php' );
```

Comment:
we can call `wp_maybe_auto_update()` directly https://github.com/WordPress/WordPress/blob/77e365efbf2e499e2ed11d29c101ea466cf1ceed/wp-includes/update.php#L493